src/System.Private.ServiceModel/tests/Stress/SharedPoolsOfWCFObjects/CallStats.cs

```diff
@@ -8,7 +8,7 @@
 using System.Threading;
 using System.Diagnostics;
 
-namespace WCFClientStressTests
+namespace SharedPoolsOfWCFObjects
```

Comment:
These 'SharedPoolsOfWCFObjects' are just some of a few other possible client stress tests.